run_cupy_install_test.py

```diff
@@ -39,7 +39,7 @@
     # make sdist
     # cuda, cudnn and numpy is required to make a sdist file.
     build_conf = {
-        'base': 'ubuntu16_py27',
+        'base': 'ubuntu18_py38-pyenv',
```

Comment:
CUDA 8.0 does not support Ubuntu 18.
```suggestion
        'base': 'ubuntu16_py35',
```

c.f. https://github.com/chainer/chainer-test/blob/fd2f22a6989a9567c12b2683ff9e8cd0b46a502f/shuffle.py#L173

Reply:
Thanks, fixed in 0c429688a6f3c49a64271b062d387ce0f0badb76.